utils/misc/dbutils.go

```diff
@@ -3,32 +3,51 @@ package misc
 import (
 	"database/sql"
 	"fmt"
+	"os"
 	"strconv"
+	"sync"
 
 	"github.com/lib/pq"
 
 	"github.com/rudderlabs/rudder-server/config"
 )
 
 var (
-	host, user, password, sslmode string
-	port                          int
+	host, user, password, dbname, sslmode, appName string
+	port                                           int
+	once                                           sync.Once
 )
 
 func loadConfig() {
 	host = config.GetEnv("JOBS_DB_HOST", "localhost")
 	user = config.GetEnv("JOBS_DB_USER", "ubuntu")
+	dbname = config.GetEnv("JOBS_DB_DB_NAME", "ubuntu")
 	port, _ = strconv.Atoi(config.GetEnv("JOBS_DB_PORT", "5432"))
 	password = config.GetEnv("JOBS_DB_PASSWORD", "ubuntu") // Reading secrets from
 	sslmode = config.GetEnv("JOBS_DB_SSL_MODE", "disable")
```

Comment:
We should consider dropping (in a backwards-compatible way) the `JOBS` prefix since we are using it for multiple functionalities of rudder-server.

Reply:
doing it in another PR here:
https://github.com/rudderlabs/rudder-server/blob/f55e59f0e92857601235ab702d447a749a1d3341/config/config.go#L271-L279